triage-procedure.md

```diff
@@ -234,3 +234,37 @@ and add tags as you see fit. The following categories should, ideally, be assign
  - If applicable, add platform tags (O-). It's fine to add more than one.
 
 [list of untagged issues]: https://github.com/rust-lang/rust/issues?utf8=%E2%9C%93&q=is%3Aissue%20is%3Aopen%20sort%3Acreated-asc%20-label%3AC-feature-request%20-label%3AC-enhancement%20-label%3AC-cleanup%20-label%3AC-bug%20-label%3AC-tracking-issue%20-label%3AC-future-compatibility%20-label%3AC-question%20-label%3AC-feature-accepted
+
+# State Of Rust Triage
+
+1. Visit the [State Of Rust] project page. Each card has three pieces of
+   information.
+     * “Feature” — The name of the feature with a link to the tracking issue.
+     * “What’s next?” — What we are waiting on to implement and stabilise
+       the RFC.
+     * “Last Update” — The last time this card has been triaged.
+2. For each card that you choose to triage:
+3. Visit the respective tracking issue, and any related issues that the tracking
+   issue is recently mentioned in.
+4. If the “What’s next?” on the card does not match what you think the current
+   state is, update it with the new information.
+5. If the implementation of an RFC has changed since the last update, move it to
+   the relevant column. 
+     * If there are PRs merged that implement the RFC the card would move
+        to “Implemented”.
+     * If there are only open PRs or the PRs don’t implement the full RFC the
+        card would be moved to “Implementation in progress”.
+     * If there has been a decision to deprecate the RFC, move that to the
+        “Deprecated” column.
+6. If there have been no meaningful changes to the RFC within 21 days,
+   ping someone for an update on the status of the feature.
+     * If there have been PRs implementing the RFC, ping the author(s).
+     * If author has not responded within a week, or there are no relevant PRs,
+       ping the relevant team.
+     * If there is no clear choice for the team that should be doing the
+       implementation, ping the team and/or individual(s) responsible for
+       the RFC.
```

Comment:
FWIW, I think this should essentially never happen. Could we add a "please add this to release team meeting notes" so we can discuss what went wrong to let this happen? Ideally all RFCs are pretty much well placed to be implemented by a concrete team(s).

Reply:
Yeah I cannot recall an instance where I made a tracking issue with no responsible team.